Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2004-04-24
Initial Package Version: 1.5.1
Origin: OpenJade SourceForge Patch Page
Upstream Status: Fixed
Description: Fixes Compile Issues with GCC 3.4
 
$LastChangedBy: bdubbs $
$Date: 2004-08-07 18:56:30 -0600 (Sat, 07 Aug 2004) $

--- OpenSP-1.5.1/include/RangeMap.cxx.orig	2004-04-24 17:39:50.000691400 +0000
+++ OpenSP-1.5.1/include/RangeMap.cxx	2004-04-24 17:39:59.682536194 +0000
@@ -7,6 +7,7 @@
 #include "RangeMap.h"
 #include "ISet.h"
 #include "types.h"
+#include "constant.h"
 
 #ifdef SP_NAMESPACE
 namespace SP_NAMESPACE {
